mindjunkies/courses/apps.py

```
@@ -3,8 +3,8 @@
 
 class CoursesConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
-    name = 'courses'
+    name = 'mindjunkies.courses'
 
 
     def ready(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `ready`

Add return type annotation: `None`

(ANN201)